### ISSUE TITLE
Scala 3.0.0 released

### ISSUE THREAD

**Issue by scf37** [scalatest/scalatest#2027]
Publish please?

**Comment by cheeseng** [comment]
@scf37 Great, thanks for informing, I'll be working on it.

**Comment by cheeseng** [comment]
@scf37 Hmm, I just done a PR and trying to build for 3.0.0, but not seeing it yet on maven: 

```
[error] sbt.librarymanagement.ResolveException: Error downloading org.scala-lang:scala3-library_3:3.0.0
[error]   Not found
[error]   Not found
[error]   not found: /home/cheeseng/.ivy2/local/org.scala-lang/scala3-library_3/3.0.0/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scala-lang/scala3-library_3/3.0.0/scala3-library_3-3.0.0.pom
[error]   not found: https://oss.sonatype.org/content/groups/public/org/scala-lang/scala3-library_3/3.0.0/scala3-library_3-3.0.0.pom
[error] 	at lmcoursier.CoursierDependencyResolution.unresolvedWarningOrThrow(CoursierDependencyResolution.scala:258)
[error] 	at lmcoursier.CoursierDependencyResolution.$anonfun$update$38(CoursierDependencyResolution.scala:227)
[error] 	at scala.util.Either$LeftProjection.map(Either.scala:573)
[error] 	at lmcoursier.CoursierDependencyResolution.update(CoursierDependencyResolution.scala:227)
[error] 	at sbt.librarymanagement.DependencyResolution.update(DependencyResolution.scala:60)
[error] 	at sbt.internal.LibraryManagement$.resolve$1(LibraryManagement.scala:59)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$12(LibraryManagement.scala:133)
[error] 	at sbt.util.Tracked$.$anonfun$lastOutput$1(Tracked.scala:73)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$20(LibraryManagement.scala:146)
[error] 	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11(LibraryManagement.scala:146)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11$adapted(LibraryManagement.scala:127)
[error] 	at sbt.util.Tracked$.$anonfun$inputChangedW$1(Tracked.scala:219)
[error] 	at sbt.internal.LibraryManagement$.cachedUpdate(LibraryManagement.scala:160)
[error] 	at sbt.Classpaths$.$anonfun$updateTask0$1(Defaults.scala:3667)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error] 	at sbt.Execute.work(Execute.scala:291)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] (scalacticDotty / update) sbt.librarymanagement.ResolveException: Error downloading org.scala-lang:scala3-library_3:3.0.0
[error]   Not found
[error]   Not found
[error]   not found: /home/cheeseng/.ivy2/local/org.scala-lang/scala3-library_3/3.0.0/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scala-lang/scala3-library_3/3.0.0/scala3-library_3-3.0.0.pom
[error]   not found: https://oss.sonatype.org/content/groups/public/org/scala-lang/scala3-library_3/3.0.0/scala3-library_3-3.0.0.pom
[error] Total time: 5 s, completed May 13, 2021 8:22:02 PM
```

It seems like we need to be a bit more patient.

**Comment by scf37** [comment]
Should work now

**Comment by cheeseng** [comment]
@scf37 Yes, available now, I am working on 3.2.9 release now.

**Comment by cheeseng** [comment]
@scf37 Fyi I have staged the 3.2.9 artifacts, @bvenners shall have the pleasure to release them.  :)

**Comment by bvenners** [comment]
FYI ScalaTest/Scalactic artifacts for 3.2.9 are out now. The corresponding ScalaTest+ artifacts are being released.

**Comment by bvenners** [comment]
All are released now!

**Comment by adamw** [comment]
@bvenners thanks, already releasing quicklens/sttp using scalatest 3.2.9 :)

**Comment by cheeseng** [comment]
Released with 3.2.9.
